bin/make_view

```diff
@@ -198,10 +198,21 @@ CREATE VIEW independent_candidate_expenditures AS
   AND "FPPC" IS NOT NULL
   AND "Cand_NamL" IS NOT NULL;
 
+DROP VIEW IF EXISTS clean_summary;
+CREATE VIEW clean_summary AS
+  SELECT * from "Summary" s
+  WHERE
+  NOT EXISTS
+    (select t."Rpt_Date" from "Summary" t where s."Filer_ID" = t."Filer_ID"
+    and t."From_Date" <= s."From_Date" and s."Thru_Date" <= t."Thru_Date"
+    and (t."From_Date" <> s."From_Date" or s."Thru_Date" <> t."Thru_Date")
+    )
```

Comment:
Maybe add a comment in the SQL about what you're trying to accomplish here since this is very thoughtful SQL here.